test/Auth/authSaga.js

```diff
@@ -0,0 +1,17 @@
+import test from 'ava';
+import { take, select } from 'redux-saga/effects';
+import { LOGIN_REQUEST } from '../../src/Auth/authModule';
+import loginFlow from '../../src/Auth/authSaga';
+
+test('loginFlow', (t) => {
+  const gen = loginFlow();
+  t.deepEqual(
+    gen.next().value,
+    take(LOGIN_REQUEST),
+  );
+
+  t.deepEqual(
+    gen.next().value,
+    select(),
+  );
+});
```

Comment:
You need another test to prove that it does get authModule from the store and use authModule.username to update localStorage